cmd/snap-confine/seccomp-support.c

```diff
@@ -45,6 +47,31 @@ static const char *filter_profile_dir = "/var/lib/snapd/seccomp/bpf/";
 
 typedef struct sock_filter bpf_instr;
 
+// keep in sync with snap-seccomp/main.go
+//
+// header of a seccomp.bin2 filter file, in network byte order (big-endian)
+//
+// Note that padding is important
+// alternatively "__attribute__((__packed__))" could be used
```

Comment:
Definitely add `__attribute__((__packed__))` for on-disk structures. While alignment here should never be an issue, there is no reason for not using packed here to ensure we control structure padding.